docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,443 @@
+
+```
+fabric-ca-client enroll -d -u https://icaadmin:icaadminpw@my-machine.example.com:7054 --tls.certfiles ../crypto/ca-cert.pem -M msp-ica --csr.hosts 'host1,*.example.com'
+```
+
+The intermediate CA TLS signing certificate is generated under `FABRIC_CA_CLIENT_HOME=tls-ca/admin/msp-ica/signcert` and the private key is available under `FABRIC_CA_CLIENT_HOME=tls-ca/admin/msp-ica/keystore`.
+
+When you deploy the intermediate CA you will need to refer to these two files in the intermediate CA configuration .yaml file.
+
+## Deploy an organization CA
+
+The deployment process overview describes the need for both an enrollment CA and a TLS CA for every organization. The TLS CA issues the TLS certificates that allow for secure transactions within the organization. The enrollment CA, also referred to as the "organization CA" or the "eCert CA" is used to issue identities for the organization. You deployed the TLS CA in the previous set of steps, now we are ready to deploy the organization CA. Later in this topic we will also create an intermediate CA; therefore, this CA serves as the "root CA" in that chain of trust.
+
+Because you've already enrolled your organization CA admin in the previous step, you are ready to deploy the CA following the same pattern of steps that were used when you deployed the TLS CA.
+
+### Before you start
+
+- Copy the Fabric CA server binary `fabric-ca-server` to a new directory on your machine.
+- Make sure you have copied the CA admin TLS certificate and key pair that you generated in the previous step to a location that can be accessed by this CA server. These are the `/msp/signcerts/cert.pem` and `/msp/keystore/` files that were generated by the enroll command in the /tls-ca folder.
+
+### Initialize the CA server
+
+Run the command to initialize the server specifying a new admin user id and password for the CA:
+
+```
+fabric-ca-server init -b <ADMIN_USER>:<ADMIN_PWD>
+```
+
+For example:
+```
+fabric-ca-server init -b rcaadmin:rcaadminpw
+```
+
+### Modify the CA server configuration
+
+As we did with the TLS CA, we will now edit the generated `fabric-ca-server-config.yaml` file for the organization CA to modify the default configuration settings for your use case according to the [Checklist for a production CA server](ca-config.html).
+
+At a minimum, you should edit the following fields:
+- `tls.enabled` - Enable TLS by setting this value to `true`.
+- `tls.certfile` and `tls.keystore`- Enter the path and filenames for the TLS CA signing certificate and private key that were generated when the CA admin for this node was enrolled with the TLS CA. The signing certificate, `cert.pem`, was generated using the Fabric CA client and can be found under `FABRIC_CA_CLIENT_HOME/msp/signcerts`. The private key is located under `FABRIC_CA_CLIENT_HOME/msp/keystore`.
```

Comment:
You have to enter the full path here, including the name of the certificate and private key. I would let folks know that.

Reply:
Changed it to:
Enter the path and filenames for the TLS CA signing certificate and private key that were generated when the bootstrap admin for this CA was enrolled with the TLS CA. The signing certificate, `cert.pem`, was generated using the Fabric CA client and can be found under `fabric-ca-client/tls-ca/admin/msp-rca/signcerts/cert.pem`. The private key is located under `fabric-ca-client/tls-ca/admin/msp-rca/keystore`. The specified path name is relative to `FABRIC_CA_HOME` therefore if you are following the folder structure that is used throughout these instructions you can simply specify `tls/cert.pem` for the `tls.certfile` and `tls/key.pem` for the `tls.keystore` or you can specify the fully qualified path name.